lib/symboldatabase.cpp

```diff
@@ -1286,7 +1286,6 @@ void SymbolDatabase::createSymbolDatabaseIncompleteVars()
     const std::set<std::string> cpp20keywords = {
         "alignas",
         "alignof",
-        "audit",
```

Comment:
As far as I know, this `audit` is a cpp20 keyword. Did you remove the wrong keyword?

Reply:
Isn't it part of concepts which did not make it in C++20?

Reply:
ok I did not know that.. then it should be removed.

the `axiom` should also be removed.

Reply:
Concepts are part of C++20, but contracts did not make it into C++20: see [Constraints and concepts](https://en.cppreference.com/w/cpp/language/constraints).

Reply:
Yes, I may have misread...

Reply:
Anyway `axiom` is not part of C++03/C++11 so it should not be in that list.